src/org/thoughtcrime/securesms/database/helpers/SQLCipherOpenHelper.java

```diff
@@ -136,6 +136,7 @@ public void onCreate(SQLiteDatabase db) {
     db.execSQL(LokiThreadDatabase.getCreatePublicChatTableCommand());
     db.execSQL(LokiUserDatabase.getCreateDisplayNameTableCommand());
     db.execSQL(LokiUserDatabase.getCreateServerDisplayNameTableCommand());
+    db.execSQL(LokiUserDatabase.getCreateProfileAvatarUrlTableCommand());
```

Comment:
you'll also have to make sure you run a migration to enable the table for existing users. Look at for `lokiv4` in the file as an example